### PR TITLE
Capstone Project entitled yayEcommerce website 

### PR DESCRIPTION
**Pull request by yayner2002**
Here I have implemented the capstone project according to the template provided. 💯 🥇 
you can find the video describing the project implementations [here](https://www.loom.com/share/de1dbbff27ff4cb8afa8cd778a97b573)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+          <ul class="navigation">
+            <li><a href="index.html" class="home">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">Blog</a></li>
+            <li><a href="#" class="contact">Buy Now</a></li>
+          </ul>
+        </div>
+      </nav>
```
Comment (zainsadaqat) [inline index.html:56]: The Mobile version of the home page isn't responsive 
- hamburger (menu) icon is too big. Kindly reduce the icon size.

Screen Shot 2022-04-15 at 4 57 35 AM

File: index.html [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+            Ethiopia's Largest <br />
+            E-commerce Site
+          </h2>
+        </section>
+        <div class="main-description">
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Architecto
+            quidem consequatur laboriosam accusantium a eveniet accusamus dicta
+          </p>
+        </div>
```
Comment (zainsadaqat) [inline index.html:75]: The Mobile version of the home page isn't responsive. I've listed down some issues. please check them out. 👍🏻 

- The subtitle text is overflowing. Kindly make it responsive.
- The heading text is overflowing. Kindly make it responsive.

Screen Shot 2022-04-15 at 5 04 25 AM


- paragraph text is too big. Kindly make it responsive and try to stick with the required design.
Screen Shot 2022-04-15 at 5 03 12 AM

File: index.html [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+        <section class="main-address">
+          <h3>Addis Ababa, Ethiopia</h3>
+          <p>@&nbsp;Akaki Kality sub city, Wereda 07, Kebele 02, Youth Hall</p>
```
Comment (zainsadaqat) [inline index.html:78]: The font size is too big. Kindly reduce the font size and make it responsive that work for both Desktop and Mobile

Screen Shot 2022-04-15 at 5 06 27 AM

File: index.html [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+            <img src="assets/images/shopping.png" alt="shopping icon" />
+            <h3>Shoes</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="service-5">
+            <img src="assets/images/shopping(1).png" alt="shopping icon" />
+            <h3>T-shirt</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </div>
+        </div>
```
Comment (zainsadaqat) [inline index.html:111]: The Mobile version of the main services section isn't responsive. Text is overflowing and hidden. Kindly fix this. 👍🏻 

Screen Shot 2022-04-15 at 5 08 30 AM

File: index.html [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+        <button class="see-whole-products" type="button">See The Whole Products</button>
```
Comment (zainsadaqat) [inline index.html:112]: The mobile version of the button isn't responsive. The font size is too big. Kindly reduce the font size that work for both Desktop and Mobile. 😃 

Screen Shot 2022-04-15 at 5 10 09 AM

File: index.html [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+          <ul>
+            <li><a href="#" id="privacy">Privacy Policy</a></li>
+            <li><a href="#">Terms of Use</a></li>
+            <li><a href="#">Accessibility</a></li>
+            <li><a href="#">Sitemap</a></li>
+          </ul>
+        </div>
+      </div>
+    </footer>
+    <script src="assets/script/script.js"></script>
```
Comment (zainsadaqat) [inline index.html:171]: In the mobile version, Our Partners and Footer section isn't showing up. Kindly fix this as well. 👍🏻 
Screen Shot 2022-04-15 at 5 14 02 AM

File: about.html [hunk 7]
```diff
@@ -0,0 +1,227 @@
...
+                class="logo"
+            /></a>
+          </div>
+          <ul class="navigation">
+            <li><a href="about.html" class="home">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">Blog</a></li>
+            <li><a href="#" class="contact">Buy Now</a></li>
```
Comment (zainsadaqat) [inline about.html]: About page should have the save navbar as in the home page to look more consistent. As you can see, On the about page the home page nav item is missing. Although you've put the home page link in the logo to look more consistent the navbar should look the same. 😄

File: about.html [hunk 8]
```diff
@@ -0,0 +1,227 @@
...
+          id fugiat non!
+        </p>
+      </div>
+      <div class="container-log">
+        <div class="about-log">
+          <img
+            src="assets/images/yaycommerceNobg.png"
+            alt="yaycommerce's logo"
+            class="logo-on-about"
+          />
```
Comment (zainsadaqat) [inline about.html:121]: This image is creating a horizontal scroll bar on the about page. Kindly make it responsive.

File: about.html [hunk 9]
```diff
@@ -0,0 +1,227 @@
...
+      <div class="container">
+        <h3 class="title">
+          YayCommerce Inc exhibition 2022 Official Logo Competition
+        </h3>
+        <p>
+          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Qui tempora,
+          necessitatibus tenetur ad modi officiis repudiandae odit quos eos,
+          deserunt mollitia? Distinctio corporis enim praesentium assumenda ab,
+          id fugiat non!
+        </p>
```
Comment (zainsadaqat) [inline about.html]: Kindly reduce the padding from its surroundings and try to stick with the original design. 👍🏻 

Screen Shot 2022-04-15 at 5 22 01 AM

File: about.html [hunk 10]
```diff
@@ -0,0 +1,227 @@
...
+          </h4>
+          <ul>
+            <li><a href="#" class="privacy">Privacy Policy</a></li>
+            <li><a href="#">Terms of Use</a></li>
+            <li><a href="#">Accessibility</a></li>
+            <li><a href="#">Sitemap</a></li>
+          </ul>
+        </div>
+      </div>
+    </footer>
```
Comment (zainsadaqat) [inline about.html]: Our partners and Footer in the mobile version aren't responsive. As you can see the logos in Our Partners section is overflowing and the footer link isn't clearly visible. Kindly make it responsive as well.

Screen Shot 2022-04-15 at 5 23 57 AM

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Hello @yayner2002 😃 😄 

You're doing a great job on your project. We're proud of you. Good job so far! 
# 👏🏻 👏🏻 👏🏻 

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

_Check the comments under the review._

[Optional]
- Please use descriptive and meaningful branch names. 👍🏻 

_If you have any confusion please feel free to let me know in the slack, Thanks!_



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by yayner2002** [comment]
here I have added the constructive comments from my reviewer. my previous reviewer asked me to add the Our Partners and Footer section In the mobile version, but according to the template, these two sections are not supposed to be shown. I don't know why he asked me to add those sections.

File: index.html (hunk 3 above)
Comment (elmar8287) [inline index.html:78]: - [x] This part of the code has to be improved: as the previous review mentioned, you need to make it responsive both Desktop and mobile, but on small screens, its display is not correct. You can see here that on small screens the text is hidden, only the `Addis Ababa` part is shown:

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @yayner2002.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by yayner2002** [comment]
here I have added a single comment given by my reviewer about responsiveness. hope you will find it interesting.
bests.

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @yayner2002 👋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**